packages/astro/src/@types/astro.ts

```diff
@@ -2331,15 +2268,14 @@ export interface AstroUserConfig {
 		 * const labradorData = await getEntry('dogs', 'labrador-retriever');
 		 * ```
 		 *
-		 * #### Rendering content
+		 * Entries generated from Markdown, MDX or Markdoc can be rendered directly to a page using the `render()` function.
 		 *
-		 * Entries generated from markdown or MDX can be rendered directly to a page using the `render()` function.
-		 *
-		 * :::caution
-		 * The syntax for rendering entries from collections that use the Content Layer is different from current content collections syntax.
+		 * :::note
+		 * The syntax for rendering collection entries is different from current content collections syntax.
 		 * :::
 		 *
 		 * ```astro
+		 * // src/pages/index.astro
```

Comment:
```suggestion
		 * ```astro title="src/pages/index.astro"
```
Both file name syntaxes work, but `.astro` files can be weird about needing to be below the `---` so I typically default to using Expressive Code's `title` property.